R/process_extract_outpatients.R

```diff
@@ -30,8 +30,6 @@ process_extract_outpatients <- function(data, year, write_to_disk = TRUE) {
       # Set smrtype variable
       smrtype = add_smr_type(.data$recid)
     ) %>%
-    # Recode GP Practice into a 5 digit number
-    # assume that if it starts with a letter it's an English practice and so recode to 99995
     dplyr::mutate(gpprac = convert_eng_gpprac_to_dummy(.data$gpprac)) %>%
```

Comment:
Just checking if we wanted to remove these notes here?

Reply:
Yep, I don't think that's needed as it's just describing what the single function does.